template/experiment/pages.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * This is the end page that is shown after the experiment
+ */
+function endPage() {
+    let link = ''
+    if (process.env.REACT_APP_redirect !== '') {
+        link = `<a href="${process.env.REACT_APP_redirect}">Back to prolific</a>`
```

Comment:
Is `prolific` always the testing tool? If not, consider making the string dynamic.

Reply:
Copied over from existing AutoRA code. Will address this in a future PR